City-of-Pori/pori-kada
# PORI-66 Added required ckeditor widget for scald integration on ckedi…

…tor, enabled picture module on ckeditor

To test:

1. build the site
2. drush updb -y && drush fra -y && drush cc all
3. drag and drop an image from scald library to body area. The image should automatically use content full responsive image style.
4. Save the page and make sure the image is full width and works responsively.
